.github/workflows/trigger-load-tests-template.yml

```diff
@@ -71,5 +75,5 @@ jobs:
                   workflow: Stdlib Workflow
                   repo: ballerina-platform/ballerina-performance-cloud
                   ref: refs/heads/main
-                  token: ${{ secrets.BALLERINA_BOT_TOKEN }}
+                  token: ${{ secrets.ballerina_bot_token }}
```

Comment:
Is this `secrets.ballerina_bot_token` or `inputs.ballerina_bot_token`?

Reply:
Also, is it safe to pass tokens using inputs like this? Shall we verify that?

Reply:
Ah never mind. This isn't an input :D

Reply:
its secrets. We provide it as an input secret